fastlane/Fastfile

```diff
@@ -1,6 +1,9 @@
 fastlane_version "2.68.0"
 default_platform :ios
 
+# The number of times the stress test suite is ran
+stress_tests_cycles = 25
+
```

Comment:
This should be local to `stress_test_v3` lane

Reply:
I thought it would be nice to have parameters like this on top of the file instead of having it all over the place.

Reply:
Also acceptable 👍 You can even pass the parameter 25 to the lane and hardcode it in the workflow file, so if you want you can easily run it with different number

Reply:
I didn't know this. What do you think is better? I think I prefer to have it in Fastfile since it's more friendly to edit. Eventually, we can extract these parameters to a standalone file.

Reply:
OK, let's keep it 👌